pallets/thea/src/lib.rs

```diff
@@ -647,8 +812,10 @@ pub mod pallet {
 			public_key: [u8; 64],
 			bit_map: u128,
 			bls_signature: [u8; 96],
-		) -> DispatchResult {
-			let _relayer = ensure_signed(origin)?;
+			_relayer_index: u8,
+			_signature: T::Signature,
```

Comment:
Why do we need these args if we are not using them?

Reply:
Yes, we are using the argument to validate the tx

Reply:
Same as [here](https://github.com/Polkadex-Substrate/Polkadex/pull/669#discussion_r1143054287)